tests/conftest.py

```diff
@@ -0,0 +1,76 @@
+import asyncio
+import random
+import string
+
+import asyncpg
+import pytest
+
+from pathlib import Path
+
+from swoop.api.config import Settings, get_settings
+
+
+@pytest.fixture(scope="session")
+def dbschema(pytestconfig) -> Path:
+    return pytestconfig.rootpath.joinpath("db", "schema.sql")
+
+
+@pytest.fixture(scope="session")
+def settings() -> Settings:
+    return get_settings()
```

Comment:
I guess this should just be
```suggestion
    return Settings()
```

Reply:
The Settings related changes should all be on `main` - maybe just a couple commits behind?